### DISCUSSION TITLE
How to disable specific column dragging (and some general feedback)

### DISCUSSION THREAD

**Discussion by omridevk** [TanStack/table#2035] [General]
First and for most, great library!!!!

I am using useResizeColumns but I have no idea how to disable a specific column from being resize. it is really really hard to understand from either the docs or the code itself, really convoluted to say the least.
Please let me know how to do so, any help will be greatly appreciated. 

I really enjoy the functionality and the end results when using this new Hooks library API, however, I have couple of issues I would like to address:
 1. The documentation is lacking or missing.
 2. I can't personally really understand how the hooks and the plugins API comes together with the core library.
3. Copy and pasting code - I just copy and paste from the examples and trying to figure it out. Which in my opinion is not a great experience, because when you wanna do a simple change it is like changing someone else's code, you have no idea what you''ll break, or change. 

To summarise, I think all of the issues above can be resolved by putting extra effort in documenting all the features before releasing them.

Regardless, the functionality it self is freaking awesome, it just understanding all the "magic" is very hard, at least for me, maybe other people have no issues with this matter.

And Thank you for a great library as it does so many things and does them perfectly(once you got it going).
Love it. +1 star from me,.
